libs/R-TypeLogic/Global/Components/AfkFrequencyComponent.hpp

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Should this file be in the Global Library ? Is it used on the client side ?

Reply:
Fixed !